commit 25d6f0762929584cb5ecd6354380a99011fbe91a (refs/changes/68/2868/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-25T13:02:45-07:00 (1 year, 11 months ago)
    
    PUBAPI-1455 cloudapi: avoid prebuilt sdcnode getting involved on non-SmartOS for local dev

diff --git a/Makefile b/Makefile
index 3878729..96706d3 100644
--- a/Makefile
+++ b/Makefile
@@ -51,14 +51,17 @@ NODE_PREBUILT_VERSION=v4.6.1
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 	NODE_PREBUILT_TAG=zone
+else
+	NPM=npm
+	NODE=node
+	NPM_EXEC=$(shell which npm)
+	NODE_EXEC=$(shell which node)
 endif
 
 
 include ./tools/mk/Makefile.defs
 ifeq ($(shell uname -s),SunOS)
 	include ./tools/mk/Makefile.node_prebuilt.defs
-else
-	include ./tools/mk/Makefile.node.defs
 endif
 include ./tools/mk/Makefile.smf.defs
 
@@ -222,8 +225,6 @@ plugins_test: provision_limits_plugin_test
 include ./tools/mk/Makefile.deps
 ifeq ($(shell uname -s),SunOS)
 	include ./tools/mk/Makefile.node_prebuilt.targ
-else
-	include ./tools/mk/Makefile.node.targ
 endif
 include ./tools/mk/Makefile.smf.targ
 include ./tools/mk/Makefile.targ
